src/ariel-os-sensors/src/measurement_unit.rs

```diff
@@ -26,6 +26,10 @@ pub enum MeasurementUnit {
     Coulomb,
     /// Decibel (dB).
     Decibel,
+    /// Decimal degrees (°).
+    DecimalDegrees,
```

Comment:
Don't know how I missed it the first time:
```suggestion
    DecimalDegree,
```